mne/viz/ica.py

```diff
@@ -130,6 +131,159 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     return fig
 
 
+def _ica_properties_skeleton():
```

Comment:
I know, sorry for that - It just constructs the layout of the figure, but I am now thinking that maybe the `axes` param in `plot_ica_properties` is unnecessary (and generates far too many input checks) and I was considering moving this `skeleton` :skull: into `plot_ica_properties` and removing the `axes` kwarg. After all - the axes are returned to the user so they can be moved around and styled further.